### PR TITLE
Feature/annotation side panel 2.6 updated

### PR DESCRIPTION
**Pull request by jeffreycwitt**
this is an update of PR #1475. I rebased the branch on top of new changes in the develop branch

### CODE REVIEW

**Comment by jeffreycwitt** [comment]
hazaah!

File: js/src/widgets/annotationsTab.js [hunk 1]
```diff
@@ -146,18 +153,18 @@
...
-                if(_this.localState().selectedList === listClicked){
-                    //_this.deselectList(listClicked);
-                    _this.eventEmitter.publish('listDeselected.' + _this.windowId, listClicked);
-                }else{
-                    //_this.selectList(listClicked);
-                    _this.eventEmitter.publish('listSelected.' + _this.windowId, listClicked);
-                }
-
-            });
+            // listItems.on('click', function(event) {
```
Comment (mejackreed) [inline js/src/widgets/annotationsTab.js]: Should these comments be removed?

File: js/src/widgets/annotationsTab.js [hunk 2]
```diff
@@ -184,8 +191,12 @@
             '<div class="annotationsPanel">',
             '<ul class="annotationSources">',
             '{{#each annotationSources}}',
-            '<li class="annotationListItem {{#if this.selected}}selected{{/if}} {{#if this.focused }}focused{{/if}}" data-id="{{this.annotationSource}}">',
-                    '<span>{{this.annotationSource}}</span>',
+            //'<li class="annotationListItem {{#if this.selected}}selected{{/if}} {{#if this.focused }}focused{{/if}}" data-id="{{this.annotationSource.chars}}">',
```
Comment (mejackreed) [inline js/src/widgets/annotationsTab.js]: Should this comment be removed?

File: spec/widgets/annotationsTab.test.js [hunk 3]
```diff
@@ -177,16 +177,16 @@ describe('AnnotationsTab', function() {
...
-      subject.localState({ selectedList: 'glub1' });
-      subject.element.find('.annotationListItem').first().click();
-      expect(this.eventEmitter.publish).toHaveBeenCalledWith('listDeselected.' + this.windowId, 'glub1');
-    });
-    it('should select an unselected item when clicked', function() {
-      subject.localState({ selectedList: 'glub2' });
-      subject.element.find('.annotationListItem').first().click();
-      expect(this.eventEmitter.publish).toHaveBeenCalledWith('listSelected.' + this.windowId, 'glub1');
-    });
+    // it('should deselect a selected item when re-clicked', function() {
```
Comment (mejackreed) [inline spec/widgets/annotationsTab.test.js]: Should these tests be rewritten to work with new functionality or be config based?

**Pull request merged**